### PR TITLE
update install instructions for Debian, Raspbian and Ubuntu

### PR DESCRIPTION
**Pull request by lsm5**
Signed-off-by: Lokesh Mandvekar <lsm5@fedoraproject.org>

### CODE REVIEW

**Comment by lsm5** [comment]
@baude @mheon @haircommander @rhatdan @TomSweeneyRedHat PTAL

**Comment by rhatdan** [comment]
LGTM

**Comment by mheon** [comment]
LGTM, good to merge once CI is green. Thanks bunches @lsm5

**Comment by lsm5** [comment]
err ... I'll change the wording a bit in one of the sections...

**Comment by lsm5** [comment]
ready to go from my side..

File: install.md [hunk 1]
```diff
@@ -10,6 +10,32 @@ sudo pacman -S podman
 
 If you have problems when running Podman in [rootless](README.md#rootless) mode follow the instructions [here](https://wiki.archlinux.org/index.php/Linux_Containers#Enable_support_to_run_unprivileged_containers_(optional))
 
+#### [Debian](https://debian.org)
+
+The [libpod](https://ftp-master.debian.org/new/libpod_1.6.4+dfsg-1.html) package is expected to be available
+soon in the default Debian repos.
```
Comment (TomSweeneyRedHat) [inline install.md]: I'm not a fan of listing a particular version or saying "soon" in a doc like this.  Two months from now when v1.7 is well entrenched, customers will be annoyed with a pointer to 1.6.4 and "soon" will be irrelevant.

I'd prefer:

```
The libpod package is expected to be available in the by Spring 2020 in the default Debian repos.  Until that time, the [libpod v1.6.4](https://ftp-master.debian.org/new/libpod_1.6.4+dfsg-1.html) is available for download.

Comment (lsm5) [reply install.md]: ack thanks, I'll update

**Comment by lsm5** [comment]
alright, so I notice there's a lot of `libpod-blah` packages in debian for perl POD:blahblah https://packages.debian.org/search?keywords=libpod&searchon=names&suite=stable&section=all and I don't see our libpod in there. I guess we gotta also include a line mentioning that libpod being unrelated to our libpod.

Also, podman is neither available in the package search results nor `apt-get install`-able yet on Debian Unstable via default repos. So, that needs to be changed too..

**Comment by rhatdan** [comment]
@lsm5 Are you taking another run at this? or should it be merged?

**Comment by lsm5** [comment]
> @lsm5 Are you taking another run at this? or should it be merged?

another run coming up..

File: install.md (hunk 1 above)
Comment (lsm5) [reply install.md]: @TomSweeneyRedHat updated the comment on official debian package. PTAL.

Comment (lsm5) [reply install.md]: I wasn't able to download binaries from the prior link, so I skipped it.

**Comment by rhatdan** [comment]
@lsm5 Needs to be rebased to fix the gating error.

**Comment by lsm5** [comment]
> @lsm5 Needs to be rebased to fix the gating error.

ready to go..

**Pull request merged**